src/templates/events/upcoming-events-template.js

```diff
@@ -28,49 +28,47 @@ const EventsList = ({ title, events }) => {
             path,
             date,
             display_date,
-            // url,
-            // fabricHosted,
           } = event.node.frontmatter;
           return (
-            <Fragment key={`${date}${title}`}>
-              <Subheading>{display_date ? display_date : date}</Subheading>
-              <Heading>
-                <Link to={path}>{title}</Link>
-              </Heading>
-              {/* {!fabricHosted && url && (
-                <Meta
+            <Fragment key={title}>
+              <Subsubheading 
+                style={{
+                  fontWeight: "500",
+                  fontSize: "1rem",
+                  margin: "0 0 0.5rem",
+                }}
+              >
+                {display_date ? display_date : date}
+              </Subsubheading>
+              <Subheading 
+                style={{
+                  fontSize: "1.4rem",
+                  margin: '0',
+                  fontWeight: "600",
+                  letterSpacing: "0.5px"
+                }}
+              >
+                <Link 
+                  to={path}
                   style={{
-                    display: "inline-flex",
-                    justifyContent: "flex-start",
-                    alignItems: "flex-start",
-                    flexDirection: isCompact ? "column" : "row",
+                    textUnderlineOffset: "0.3rem",
```

Comment:
please don't do this. style the base component. text links should have consistent styling.